offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectViewModel.groovy

```diff
@@ -118,7 +118,10 @@ class CreateProjectViewModel {
     }
 
     private void resetModel() {
+        String previousSpace = new String(resultingSpaceName)
         initFields()
+        resultingSpaceName = previousSpace
+        desiredSpaceName = previousSpace
```

Comment:
Nit: Nice solution, down the line this could be made more distinct if additional field values have to be kept (E.g. with a keepFieldValuesAfterReset Method)